canonical-web-and-design/canonicalwebteam.discourse
# Added new docs parser

## Done

- Added new `Docs` and `DocParser` classes (https://github.com/canonical-web-and-design/canonicalwebteam.discourse/issues/60)
- Added Navigation template (https://github.com/canonical-web-and-design/canonicalwebteam.discourse/issues/61)

## QA

1. In any proyect update the `requirements.txt` to contain this branch code:
```
git+git://github.com/jkfran/canonicalwebteam.discourse@new-docs-parser#egg=canonicalwebteam.discourse
```

2. Update a Discourse topic to include the new navigation table.

3. See it working

### Pair with me to see a real example working:
![image](https://user-images.githubusercontent.com/6353928/102061748-bf3d3300-3deb-11eb-9d26-9e0500746e6a.png)
### Result:
![image](https://user-images.githubusercontent.com/6353928/102061796-ccf2b880-3deb-11eb-86b8-943452734954.png)
